### PR TITLE
[modbus.e3dc] battery capacity added

### LINKED ISSUE #13171: [modbus.e3dc] Battery Storage Support
The E3DC Powerplant comes per default with an amount of local battery storage. Currently only SoC and power charging / uncharging states are submitted without knowledge of the battery storage size.

Request:
- provide configuration item with local installed battery storage capacity
- provide channels showing energy levels (charged / uncharged)

### PR DESCRIPTION
**Pull request by weymann**
Resolves #13171

### CODE REVIEW

File: bundles/org.openhab.binding.modbus.e3dc/src/main/resources/OH-INF/thing/power-channel-types.xml [hunk 1]
```diff
@@ -75,4 +75,16 @@
 		<description>Charge Level of your attached Battery in Percent</description>
 		<state pattern="%d %unit%" readOnly="true"/>
 	</channel-type>
+	<channel-type id="battery-charged-channel">
+		<item-type>Number:Energy</item-type>
+		<label>Battery Charge</label>
+		<description>Charged energy of battery</description>
+		<state pattern="%.3f %unit%" readOnly="true"/>
```
Comment (fwolter) [inline bundles/org.openhab.binding.modbus.e3dc/src/main/resources/OH-INF/thing/power-channel-types.xml:82]: Are you sure 3 positions after the decimal point are reasonable for a home storage battery? Same for below.

**Review by fwolter — CHANGES_REQUESTED** [review]
You could update the English18n properties file to make the new channels available for translation in Crowdin by running `mvn org.openhab.core.tools:i18n-maven-plugin:3.4.0:generate-default-translations`.

File: bundles/org.openhab.binding.modbus.e3dc/src/main/resources/OH-INF/thing/power-channel-types.xml (hunk 1 above)
Comment (weymann) [reply bundles/org.openhab.binding.modbus.e3dc/src/main/resources/OH-INF/thing/power-channel-types.xml:82]: From technical point I would say yes. Values are delivered in kW/h so 3 decimal points representing Watt level. 
E3DC comes with different battery sizes and smallest is 6.5 kW/h
If 1% is charged channel delivers 0,065 kW/h aka 65 W/h

Comment (fwolter) [reply bundles/org.openhab.binding.modbus.e3dc/src/main/resources/OH-INF/thing/power-channel-types.xml:82]: Sounds reasonable!

**Comment by fwolter** [comment]
Can you comment on updating the i18n file?

**Comment by weymann** [comment]
Missed i18n - added!

**Review by fwolter — APPROVED** [review]

**Comment by jlaur** [comment]
@weymann - update instructions are missing for the added channels - see https://next.openhab.org/docs/developer/bindings/thing-xml.html#updating-thing-types

**Pull request merged**